scripts/enforce_licence.py

```diff
@@ -232,11 +232,15 @@ def GetDirectoryLicences(root_dir):
     subdirs[:] = [d for d in subdirs if not d.startswith('.')]
 
     if LICENCE_FILE in files:
-      f = open(os.path.join(dir_name, LICENCE_FILE))
+      licence_path = os.path.join(dir_name, LICENCE_FILE)
+      f = open(licence_path)
       lines = f.readlines()
       f.close()
       licences[dir_name] = TransformLicence(lines)
-      print('Found LICENCE for directory %s' % dir_name)
+      rel_licence_path = "./" + os.path.relpath(licence_path)
+      rel_dir_path = "./" + os.path.relpath(dir_name)
```

Comment:
This smells a bit fishy, surely we can do something more pythonic and stable to get the relative path than just prepending ./?

Reply:
I don't think there is a more Pythonic way; I wanted to match the relative dirs starting with `./` for consistency with the other tasks. I'll take them out.

Reply:
Fixed!

Reply:
I think doing `os.path.join(".", os.path.relpath(licence_path))` would do what you intended in a slightly less fishy way (ideally with a comment explaining why), but if it's not needed that's even better!